<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FoodReserve</class>
 <widget class="QWidget" name="FoodReserve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>WelCome</string>
   </property>
  </widget>
  <widget class="QLabel" name="Name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Meigu</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lazania</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Steak</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Juje</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Khaviar</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Makaroni</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Biff</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fesenjan</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Olovie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Shenitsel</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Kabab</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Omlet</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mojudi:</string>
   </property>
  </widget>
  <widget class="QLabel" name="MojudiLable">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="NextWeek">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Week</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Submit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string> Shanbe</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string> 1 Shanbe</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string> 2 Shanbe</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string> 3 Shanbe</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string> 4 Shanbe</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string> 5 Shanbe</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Self</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Khabgah</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_8">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Self</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Khabgah</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_9">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Self</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Khabgah</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_10">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Self</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Khabgah</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_11">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Self</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Khabgah</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_12">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Self</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Khabgah</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gheimat: (Tuman)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mahale Tahvil</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Settings">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Go To Settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>3000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>150</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>4000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>180</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>6000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>240</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1500</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>210</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>4000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>270</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLable">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>You do not have enough money in your account!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Info</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>22</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>23</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>25</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>26</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>27</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="m">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>m</string>
   </property>
  </widget>
  <widget class="QLabel" name="d">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>d</string>
   </property>
  </widget>
  <widget class="QLabel" name="y">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>120</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>**</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_31">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>**</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>180</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>**</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_33">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>210</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>**</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_34">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>240</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>**</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_35">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>270</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>**</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>NOTICE :  If there is a ** sign beside the price, it means the price will be increased by</string>
   </property>
  </widget>
  <widget class="QLabel" name="perc">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>420</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="perc_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>420</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
